app/src/main/java/com/google/android/stardroid/search/PrefixStore.java

```diff
@@ -14,11 +14,7 @@
 
 package com.google.android.stardroid.search;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.*;
```

Comment:
Sounds good.

On Sun, 10 Apr 2016 at 21:10 tricao <notifications@github.com> wrote:

> In app/src/main/java/com/google/android/stardroid/search/PrefixStore.java
> <https://github.com/sky-map-team/stardroid/pull/100#discussion_r59146112>:
>
> > @@ -14,11 +14,7 @@
> >
> >  package com.google.android.stardroid.search;
> >
> > -import java.util.Collection;
> > -import java.util.Collections;
> > -import java.util.HashMap;
> > -import java.util.HashSet;
> > -import java.util.Set;
> > +import java.util.*;
>
> Done.
>
> Maybe a link in README to:
> https://google.github.io/styleguide/javaguide.html?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/sky-map-team/stardroid/pull/100/files/13eab6078ef0735924600fc8ba29697219c1b71f#r59146112>
>